runtime/lib/capi/RuntimeCAPI.cpp

```diff
@@ -163,20 +163,12 @@ void __quantum__qis__Gradient(int64_t numResults, /* results = */...)
     va_list args;
     va_start(args, numResults);
     for (int64_t i = 0; i < numResults; i++) {
-        auto *mrp = va_arg(args, ResultType *);
+        MemRefT<double, 1> *mrp = va_arg(args, ResultType *);
```

Comment:
Variable name is a bit too short for my taste 😅 What does mrp mean?

Reply:
I didn't name it. I just didn't change the name. I believe it stands for MemRefPointer.

Reply:
Fair enough but would be a good opportunity to change it :)

Reply:
[55530bb](https://github.com/PennyLaneAI/catalyst/pull/63/commits/55530bb71963adfcd964b65304b34574c34b1155)